tarsier-service/next/src/components/apiLogsTable.tsx

```diff
@@ -0,0 +1,55 @@
+import { useEffect, useState } from "react"
+import {
+  Table,
+  TableBody,
+  TableCell,
+  TableHead,
+  TableHeader,
+  TableRow,
+} from "@/components/ui/table"
+
+const BACKEND_BASE_URL = "http://localhost:8000" //import
```

Comment:
should steal the env management of create-t3-app (Can look at AgentGPT for this as well)

Basically it defines both a client and server env (though we don't really need the server env) and fails startup/deployment if required values aren't set